### ISSUE TITLE
Distance - Map preview in confirmation page does not show offline message in offline mode

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#35517]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v1.4.34-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://expensify.testrail.io/index.php?/tests/view/4256083
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause- Internal Team
**Slack conversation:**

## Action Performed:
1. Go offline.
2. Go to workspace chat.
3. Open distance request page.
4. Note that the map preview show offline message.
5. Select two addresses and proceed to confirmation page.

## Expected Result:
The map preview will show offline message.

## Actual Result:
The map preview does not show offline message. It only shows an icon.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/93399543/587eee81-e417-4c83-91b9-279c0ac825d9

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by kbecciv** [comment]
We think that this bug might be related to #vip-vsb
CC @quinthar

**Comment by Tony-MK** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

Distance - Map preview in confirmation page does not show offline message in offline mode

### What is the root cause of that problem?

In the [`MoneyRequestConfirmationList`](https://github.com/Expensify/App/blob/main/src/components/MoneyRequestConfirmationList.js#L603) component, we only show the `ComfirmedRoute` component.

https://github.com/Expensify/App/blob/ed95bc03c14e81d6ea78508b2d318ac5231d9b97/src/components/MoneyRequestConfirmationList.js#L603-L606

### What changes do you think we should make in order to solve the problem?


Just like the `DistanceEReceipt`, we use `isOffline` to render the `PendingMapView` instead of the `ComfirmedRoute` component.

**Comment by paultsimura** [comment]
This is by design:

https://github.com/Expensify/App/pull/26836#pullrequestreview-1620574101

> Please render the pending map view without any text

https://github.com/Expensify/App/pull/26836#issuecomment-1716398756

> Let's have only the icon on the confirmation page, and keep that descriptive text on the distance request page.

cc: @neil-marcellini to confirm.

**Comment by DylanDylann** [comment]
@paultsimura Thanks for your comment. It seems this is expected.

**Comment by DylanDylann** [comment]
@neil-marcellini Could you help to confirm this again? Just in case we change the design

**Comment by dylanexpensify** [comment]
Bump @neil-marcellini 🙇‍♂️

**Comment by dylanexpensify** [comment]
Bump 🙏

**Comment by dylanexpensify** [comment]
@neil-marcellini 🙈

**Comment by DylanDylann** [comment]
@dylanexpensify This issue is waiting for a [confirmation](https://github.com/Expensify/App/issues/35517#issuecomment-1920029649) from the internal engineer

**Comment by dylanexpensify** [comment]
dm'd neil!

**Comment by neil-marcellini** [comment]
Yep, our design team decided that we don't need to show another explanatory message there and we can just show an icon on the confirmation page.
